test/test_functional.py

```diff
@@ -31,6 +32,10 @@ class TestFunctional(unittest.TestCase):
     specgram = torch.tensor([1., 2., 3., 4.])
 
     test_dirpath, test_dir = common_utils.create_temp_assets_dir()
+
+    test_filepath_sinewave = os.path.join(test_dirpath, "assets", "sinewave.wav")
```

Comment:
I don't think a sinewave is enough as a test example. There should be a few others that more generic. Sinewaves don't cover a broad, mixes range of frequencies.

Reply:
We have other asset files we can use instead which I'd say would be good enough.